src/libYARP_OS/src/Node.cpp

```diff
@@ -549,13 +549,22 @@ Node::Node() :
 Node::Node(const ConstString& name) :
         mPriv(new Helper)
 {
+    ConstString _name;
+    if(name.at(0) != '/')
+    {
+        _name == "/"+name;
```

Comment:
Ok as a bugfux.
As a personal taste, I don't like so much when the the code changes user params on its will, and I think a constructor should never attempt operation which may fail, like opening port or check config params.

I propose to remove this option and substitute with (if not yet existing) an empty constructor and a init/config method which accepts the node name. This way the method can clearly fail and a better error handling can be achieved.

Can we chat a bit about this in the next IFI day?